.travis.yml

```diff
@@ -29,17 +29,17 @@ env:
       COVERAGE=true
     # This environment tests the oldest supported anaconda env
     - DISTRIB="conda" PYTHON_VERSION="2.7" INSTALL_MKL="false"
-      NUMPY_VERSION="1.6.2" SCIPY_VERSION="0.11.0" CYTHON_VERSION="0.23"
+      NUMPY_VERSION="1.8.1" SCIPY_VERSION="0.13.3" CYTHON_VERSION="0.23"
       COVERAGE=true
     # This environment tests the newest supported anaconda env
     # It also runs tests requiring Pandas.
     - DISTRIB="conda" PYTHON_VERSION="3.6" INSTALL_MKL="true"
-      NUMPY_VERSION="1.11.2" SCIPY_VERSION="0.18.1" PANDAS_VERSION="0.19.1"
+      NUMPY_VERSION="1.12.1" SCIPY_VERSION="0.18.1" PANDAS_VERSION="0.19.1"
```

Comment:
Are these numpy/scipy reqs pinned to the latest [Continuum release](https://docs.continuum.io/anaconda/pkg-docs), or the newest official releases of both libraries available _through_ anaconda?

Maybe this isn't important, but the latest Anaconda install has numpy=1.11.3, scipy=0.18.1, pandas=0.19.2.